internal/web/middleware/recoverer.go

```diff
@@ -0,0 +1,26 @@
+package middleware
+
+import (
+	aulogging "github.com/StephanHCB/go-autumn-logging"
+	"github.com/eurofurence/reg-room-service/internal/web/common"
+	"net/http"
+	"runtime/debug"
+)
+
+func PanicRecoverer(next http.Handler) http.Handler {
+	fn := func(w http.ResponseWriter, r *http.Request) {
+		defer func() {
+			rvr := recover()
+			if rvr != nil && rvr != http.ErrAbortHandler {
```

Comment:
errors.Is() is the better Go coding practice

Reply:
return type of recover() is `any`, not `error`, so I'd need to add a typecast with ok check first. Is this really best practice? This seems more readable.